go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/kyma-project/cli
 go 1.14
 
 replace (
-	github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.6.0
```

Comment:
Replace from the `Sirupsen/logrus` with the `sirupsen/logrus` is not necessary?

Reply:
There is no usage of "old" `Sirupsen` in module: 
https://github.com/kyma-project/cli/blob/0099ed464748d9a71ab722ac7e9ec67446b02de8/go.sum#L1015-L1021